src/FSharp.Control.TaskSeq/TaskExtensions.fs

```diff
@@ -7,9 +7,9 @@ open System.Threading.Tasks
 open Microsoft.FSharp.Core.CompilerServices
 open Microsoft.FSharp.Core.LanguagePrimitives.IntrinsicOperators
 
-#nowarn "57"
-#nowarn "1204"
-#nowarn "3513"
+// note: these are *not* experimental features anymore, but they forgot to switch off the flag
+#nowarn "57" // Experimental library feature, requires '--languversion:preview'.
```

Comment:
```suggestion
#nowarn "57" // Experimental library feature, requires '--langversion:preview'.
```

Reply:
you're so quick! I noticed it, thanks :)